apps/projects/app/components/Panel/NewIssue/NewIssue.js

```diff
@@ -100,17 +100,11 @@ class NewIssue extends React.PureComponent {
 
     const id = selectedProject > 0 ? reposIds[selectedProject - 1] : ''
 
-    const reGet = [
-      {
-        query: GET_ISSUES,
-        variables: { reposIds },
-      },
-    ]
+    // TODO: refetch Issues list after mutation
 
     return (
       <Mutation
         mutation={NEW_ISSUE}
-        refetchQueries={reGet}
```

Comment:
It seems like something could be getting out-of-sync here by looking at the message (the stacktrace says it is happening on the new issue mutation)
![image](https://user-images.githubusercontent.com/5030059/60846898-34610d80-a1e1-11e9-87ab-541c17535815.png)

Reply:
This happens right after clicking the new issue creation button at the new issue panel

Reply:
Is that related to the `refetchQueries` prop? 

From what I could tell, we were originally fetching issues with this `GET_ISSUES` query. That would have made it so that when we called `refetchQueries`, Apollo cleared its cache for that query, refetched, and we got the newly-created issue to show up right away. However, when we added pagination, with the more complicated query, this `refetchQueries` prop never got updated, and the functionality broke.

Last I knew (early this year), expiring the cache for these sorts of complicated queries is more complex than you'd expect, with Apollo. That's why I put in a TODO instead of just updating it.

However, I'm not sure any of this would be related to the error that you're seeing.